docs/docs/commands.md

```diff
@@ -12,24 +12,25 @@ meta:
 _**Key:**_  
 `< >` = Required  
 `[ ]` = Optional  
+`|` = Means "or"  
 `bot` = Refers to the Suggestions bot  
 
 ## General Commands
 | Command        | Description           | Guide  |
 | ------------- |:-------------:| -----:|
 | changelog      | View the recent changelog for the bot | [click](#changelog-command) |
-| channel | View the current suggestions channel in the guild | [click](#channel-command) |
+| channel | View the current suggestions channel in the server | [click](#channel-command) |
 | help      | View bot commands and where to receive support      |   [click](#help-command) |
 | info | View bot information      |    [click](#info-command) |
 | invite | Receive a DM with information on inviting the bot to your server | [click](#invite-command) | 
-| patrons | View all Patrons who have pledged to Nerd Cave Development | [click](#patrons-command) |
+| patrons | View all Patrons who've pledged to Nerd Cave Development | [click](#patrons-command) |
```

Comment:
This command no longer exists to the best of my knowledge.

Reply:
Thanks for this information